WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -647,6 +647,18 @@ protected void onCreate(Bundle savedInstanceState) {
         setupPreviewUI();
     }
 
+    private void presentNewPageNoticeIfNeeded() {
+        if (mIsPreview
+            || !mIsPage
+            || !mModalLayoutPickerFeatureConfig.isEnabled()
+            || !mEditPostRepository.isLocalDraft()) {
```

Comment:
Could you please double check we really want to use `!mEditPostRepository.isLocalDraft()` here? `Local draft` means that the post is a draft and has not been synced with the server yet. I tried to reproduce an issue with this condition and I think I was successful:

1. Click on Create a Page
2. Select a layout
3. Click on "Create Page"
4. Notice "Page Created" message (this is expected)
5. Turn on airplane mode
6. Leave the editor
7. Go to "Page list"
8. Open the page from step 3
9. Notice" Page created" message (this is imo not expect)


Perhaps the `mIsNewPost` field in the EditPostActivity might work for us. Wdyt?

Reply:
Thank you for reviewing this @malinajirka 🙇 
Nice catch 👍 
Fixed with 80ba44fe8dd2945acb477f84c1bc487eef7510c5